specification/cognitiveservices/data-plane/TranslatorText/Batch/preview/v1.0/TranslatorBatch.json

```diff
@@ -0,0 +1,1469 @@
+          "format": "int32",
+          "description": "Total count",
+          "type": "integer",
+          "example": 1
+        },
+        "failed": {
+          "format": "int32",
+          "description": "Failed count",
+          "type": "integer",
+          "example": 0
+        },
+        "success": {
+          "format": "int32",
+          "description": "Number of Success",
+          "type": "integer",
+          "example": 1
+        },
+        "inProgress": {
+          "format": "int32",
+          "description": "Number of in progress",
+          "type": "integer",
+          "example": 0
+        },
+        "notYetStarted": {
+          "format": "int32",
+          "description": "Count of not yet started",
+          "type": "integer",
+          "example": 0
+        },
+        "cancelled": {
+          "format": "int32",
+          "description": "Number of cancelled",
+          "type": "integer",
+          "example": 0
+        }
+      }
+    },
+    "OperationStatusSummary": {
+      "description": "Job status response",
+      "required": [
+        "createdDateTimeUtc",
+        "id",
+        "lastActionDateTimeUtc",
+        "summary"
+      ],
+      "type": "object",
+      "properties": {
+        "id": {
+          "format": "uuid",
+          "description": "Id of the operation.",
+          "type": "string",
+          "example": "36724748-f7a0-4db7-b7fd-f041ddc75033"
+        },
+        "createdDateTimeUtc": {
+          "format": "date-time",
+          "description": "Operation created date time",
+          "type": "string",
+          "example": "2020-01-20T11:40:07.7010000-08:00"
+        },
+        "lastActionDateTimeUtc": {
+          "format": "date-time",
+          "description": "Date time in which the operation's status has been updated",
+          "type": "string",
+          "example": "2020-01-20T11:41:07.7010000-08:00"
+        },
+        "status": {
+          "$ref": "#/definitions/Status"
+        },
+        "summary": {
+          "$ref": "#/definitions/StatusSummary"
+        }
+      }
+    },
+    "OperationStatusResponse": {
+      "description": "Document Status Response",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "The summary status of individual operation",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/OperationStatusSummary"
+          }
+        },
+        "@nextLink": {
+          "description": "Url for the next page.  Null if no more pages available",
+          "type": "string",
+          "example": "https://westus.cognitiveservices.azure.com/translator/text/batch/v1.0.preview.1/operations?$top=5&$skip=15"
+        }
+      }
+    },
+    "DocumentSubmissionResponse": {
+      "description": "The response for a document submission",
+      "type": "object",
+      "properties": {
+        "job": {
+          "$ref": "#/definitions/OperationStatusSummary"
```

Comment:
Since the user will also have the Location and Operation-Location headers, it may make sense to remove the OperationStatusSummary here.

Reply:
make sense.

Reply:
removed.